### ISSUE TITLE
Search Bar Autocomplete and Screen Reader(s)

### ISSUE THREAD

**Issue by mtribone** [psu-libraries/cho#713]
Putting a ref to https://github.com/projectblacklight/blacklight/issues/1907 so that I remember to fix this upstream.

**Comment by mtribone** [comment]
Do we want autocomplete? And if so do we want the autocomplete provided by Blacklight if it is fixed? Or do we want to implement autocomplete in another way? 

Related https://github.com/projectblacklight/blacklight/issues/1907
